gsa/src/gmp/models/cve.js

```diff
@@ -39,10 +39,14 @@ class Cve extends Info {
     } else if (hasValue(ret.cvssV2Vector)) {
       ret.vector = ret.cvssV2Vector;
     } else {
-      ret.vector = null;
+      ret.vector = '';
```

Comment:
Shouldn't we keep that?
```suggestion
      ret.vector = undefined;
```